src/prof_stack_range.c

```diff
@@ -13,6 +13,13 @@
 #include <string.h>
 #include <unistd.h>
 
+// Fix undeclared function 'gettid' on Ubuntu18
```

Comment:
What would be the best way to have this on Ubuntu18 or on any other platforms that might fail? I haven't tested it on other platforms.